### ISSUE TITLE
Create a blog post for "Mind the product" to cover "How to open source a project"

### ISSUE THREAD

**Issue by stichbury** [kedro-org/kedro-devrel#82]
Could potentially align with, or be close to, either our 4th anniversary or other things that are in-progress (details redacted)

https://www.mindtheproduct.com/write-for-mind-the-product/

**Comment by astrojuanlu** [comment]
We missed the deadline but we can target our 5th anniversary or whenever we reach 10k GH stars :)

**Comment by stichbury** [comment]
Yes this could be an interesting one to do with Vizro too, so I'l look into it.
